src/impl/util.js

```diff
@@ -203,3 +203,11 @@ export function normalizeObject(obj, normalizer, ignoreUnknown = false) {
 export function timeObject(obj) {
   return pick(obj, ['hour', 'minute', 'second', 'millisecond']);
 }
+
+export const customInspectSymbol = (() => {
+  try {
+    return require('util').inspect.custom; // eslint-disable-line global-require
+  } catch (_err) {
+    return Symbol('util.inspect.custom');
```

Comment:
Can you explain this? I'm a little out of the loop on this stuff

Reply:
Oh, sure.

The recommended way for custom routine during `inspect` is to specify a symbol, which is exposed as `util.inspect.custom`.

However, for browser environments, where `require` is unavailable, we need a clean no-op. That is handled in `catch` clause: `util.inspect.custom` is just for show, it is merely a description, could be anything. We simply return a new symbol (that won't be used by anyone) so we can simply do

```js
import { customInspectSymbol } from 'utils';

[customInspectSymbol]() {
  // ...
}
```

in our classes, without any helpers or extra checks.

Reply:
Got it